pkg/registry/apis/provisioning/jobs/legacy/types.go

```diff
@@ -0,0 +1,33 @@
+package legacy
+
+import (
+	"database/sql"
+	"time"
+
+	"github.com/go-git/go-git/v5/plumbing/object"
+)
+
+type userInfo struct {
+	name  string
+	email string
+	login string
+}
+
+func (u *userInfo) getAuthor() object.Signature {
+	return object.Signature{
+		Name:  firstRealStringX(u.name, u.login, u.email, "?"),
+		Email: firstRealStringX(u.email, u.login, u.name, "?"),
+		When:  time.Now(),
+	}
+}
+
+type folderInfo struct {
```

Comment:
[style] This can be moved to `export_folders.go` and this file removed. It's only used there.